### ISSUE TITLE
Allow query strings spread into multiple lines

### ISSUE THREAD

**Issue by NikosEfthias** [Huachao/vscode-restclient#16]
- VSCode Version:1.3.1
- OS Version:ubuntu 16.4
- REST Client Version:0.6.1

Steps to Reproduce:

 some queries to api are really lng and writing them in single line is making everything worse
it could be so nice to split parts into lines but when i do this the request wont work

**Comment by Huachao** [comment]
@NikosEfthias , I am considering this, and since I am busy these days, I will implement this ASAP, sorry for the inconvenience.

**Comment by NikosEfthias** [comment]
awesome thank you :+1:

**Comment by Huachao** [comment]
@NikosEfthias, when I think about this feature recently, I found that allowing request cross over multiple lines will have a problem that syntax highlight not working. And I think one simple way to this is just word wrap in Visual Studio Code by `View->Toggle Word Wrap`.

**Comment by NikosEfthias** [comment]
wordwrap is great but it does not allow you to break from the place you wish

**Comment by Huachao** [comment]
@NikosEfthias good point, I know your pain, and I will consider if implementing this feature will affect others, and will update in this thread ASAP

**Comment by NikosEfthias** [comment]
as i told a really long get query each part after ? could be in a new line terminated by `\` so it could be really easy to edit or change values

**Comment by Huachao** [comment]
@NikosEfthias agree

**Comment by NikosEfthias** [comment]
at the moment it is really pain to horizontal scroll for changing something and even more difficult when its half of the screen

**Comment by Huachao** [comment]
@NikosEfthias I'd like to __ONLY__ spread query string into multiple lines
Following requests will be equivalent:
```http
GET https://example.org/items?name=hello&type=world
```
or
```http
GET https://example.org/items
?name=hello
&type=world
```
or

```http
GET https://example.org/items
?name=hello&type=world
```

**Comment by NikosEfthias** [comment]
```http
GET https://example.org/items
?name=hello
&type=world
```
this is way more clear

**Comment by Huachao** [comment]
@NikosEfthias I have 
Following requests will be equivalent:
```http
GET https://example.org/items?name=hello&type=world
```
or
```http
GET https://example.org/items
?name=hello
&type=world
```

**Comment by NikosEfthias** [comment]
when is the update gonna be ready ?

**Comment by Huachao** [comment]
@NikosEfthias it will in one or two days, since I am adding another feature, and will publish the new version together with that feature. Sorry for the inconvenience

**Comment by Huachao** [comment]
@NikosEfthias I have published a new version 0.7.4, you can try to verify. Thanks
